apis/Google.Bigquery.V2/Google.Bigquery.V2.Snippets/BigqueryClientSnippets.cs

```diff
@@ -647,6 +648,600 @@ public void ParameterizedQuery_PositionalParameters()
+            // report errors etc.
+            // End snippet
+
+            var result = job.PollUntilCompleted();
+            // If there are any errors, display them *then* fail.
+            if (result.Status.ErrorResult != null)
+            {
+                foreach (var error in result.Status.Errors)
+                {
+                    Console.WriteLine(error.Message);
+                }
+            }
+            Assert.Null(result.Status.ErrorResult);
+
+            int rowsAfter = table.ListRows().Count();
+            Assert.Equal(rowsBefore + 2, rowsAfter);
+        }
+
+        [Fact]
+        public async Task CreateQueryJobAsync()
+        {
+            var projectId = _fixture.ProjectId;
+            var datasetId = _fixture.GameDatasetId;
+            var historyTableId = _fixture.HistoryTableId;
+            var queryTableId = Guid.NewGuid().ToString().Replace('-', '_');
+
+            // Snippet: CreateQueryJobAsync(*,*)
+            BigqueryClient client = await BigqueryClient.CreateAsync(projectId);
+            BigqueryTable table = await client.GetTableAsync(datasetId, historyTableId);
+            TableReference destination = client.GetTableReference(datasetId, queryTableId);
+            // If the destination table is not specified, the results will be stored in
+            // a temporary table.
+            BigqueryJob job = await client.CreateQueryJobAsync(
+                $@"SELECT player, MAX(score) AS score
+                   FROM {table}
+                   GROUP BY player
+                   ORDER BY score DESC",
+                new CreateQueryJobOptions { DestinationTable = destination });
+
+            // Wait for the job to complete.
+            await job.PollUntilCompletedAsync();
+
+            // Then we can fetch the results, either via the job or by accessing
+            // the destination table.
+            BigqueryQueryJob result = await client.GetQueryJobAsync(job.Reference);
+            await result.GetRowsAsync().ForEachAsync(row =>
+            {
+                Console.WriteLine($"{row["player"]}: {row["score"]}");
+            });
+            // End snippet
+
+            var players = result.GetRows().Select(r => (string)r["player"]).ToList();
+            Assert.Contains("Ben", players);
+            Assert.Contains("Nadia", players);
+            Assert.Contains("Tim", players);
+        }
+
+        [Fact]
+        public async Task ListJobsAsync()
+        {
+            var projectId = _fixture.ProjectId;
+
+            // Snippet: ListJobsAsync(*,*)
+            BigqueryClient client = await BigqueryClient.CreateAsync(projectId);
+            var jobs = await client.ListJobsAsync().Take(20).ToList();
+            foreach (var job in jobs)
+            {
+                Console.WriteLine(job.Reference.JobId);
+            }
+            // End snippet
+
+            Assert.NotEmpty(jobs);
```

Comment:
No, but setting up the fixture does - so we'll definitely have some jobs to list here.